core/creators/themedui.js

```diff
@@ -569,3 +638,48 @@ CKEDITOR.config.startupMode = 'wysiwyg';
  * @param {String} data.space The name of the {@link CKEDITOR.ui#space space} for which the event is fired.
  * @param {String} data.html HTML string which will be included in the given space.
  */
+
+/**
+ * Allow delay editor creation if target element is detached from DOM.
+ *
+ *		config.delayDetached = true;
+ *
+ * @cfg {Boolean} [delayDetached=false]
+ * @member CKEDITOR.config
+ */
+CKEDITOR.config.delayDetached = false;
+
+/**
+ * Verifying frequency (ms) to use with default {@link CKEDITOR.config#registerCallback callback}.
+ *
+ * Used to set `setInterval` which checks whenever element is attached to DOM again.
+ *
+ *		config.delayDetachedFrequency = 1000;
+ *
+ * @cfg {Number} [delayDetachedFrequency=50]
+ * @member CKEDITOR.config
+ */
+CKEDITOR.config.delayDetachedFrequency = 50;
+
+/**
+ * Function with single argument. As argument is passed another function that continues editor creation.
+ * Allows to store create function and invoke it when it is convenient instead of periodically invoke element verifying.
+ *
+ *		// Default value.
+ *		config.registerCallback = function ( func ) {
+ *			func();
+ *		};
+ *
+ *		// Possible use:
+ *		// Save creation callback.
+ *		// Call resumeEditorCreation whenever you choose.
+ *		config.registerCallback = function ( createEditor ) {
+ *			resumeEditorCreation = createEditor;
+ *		};
+ *
+ * @cfg {Function} [registerCallback=see example]
```

Comment:
Such a reference doesn't look great in docs TBH: https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-blockedKeystrokes. Let's pass an actual default here (if multiline comment doesn't work here, it's short enough to inline it - `function( callback ) { callback() };`).